stockflux-securities-master/src/components/SecuritiesTable.css

```diff
@@ -76,7 +76,7 @@
   border-left: 1px solid var(--clr-divider-light);
 }
 
-.securities-error-message,
+.securities-error-message-container,
```

Comment:
I would rename this to something like `message-container` since its purpose is not *just* to contain error messages.

Reply:
In the securities table it is currently only being used to contain error messages. I renamed it because there was a css conflict in the message component I forgot to remove.

Reply:
Changed the name now